lib/mixlib/install/generator/powershell/scripts/helpers.ps1.erb

```diff
@@ -120,7 +120,7 @@ function Test-ProjectPackage {
   param ($Path, $Algorithm = 'SHA256', $Hash)
   if (!$env:Valid_ProjectPackage){
     Write-Verbose "Testing the $Algorithm hash for $path."
-    $ActualHash = if (Is-FIPS) {
+    $ActualHash = if ($(Is-FIPS) -ge 1 -or $(Get-PowershellVersion) -le 3) {
```

Comment:
Hi @mwrock - I have verified only with `Custom-GetFileHash` and deploy multiVMs (numberofinstances = 10) using azurerm template and that's working for me but still I am thinking `Get-FileHash` cmdlet is good due to in `Custom-GetFileHash` method many loop between `disposable` and `use` function.

So Please confirm should we only use `Custom-GetFileHash` for all cases?